File: src/main/java/com/galaxy/Restaurantinformationsystem/entity/MenuEntity.java
package com.galaxy.Restaurantinformationsystem.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class MenuEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private Long price;
    private String image;

    @ManyToOne
    private StoreEntity store;
}

File: src/main/java/com/galaxy/restaurant/information/system/entity/MenuEntity.kt
package com.galaxy.restaurant.information.system.entity

import jakarta.persistence.*

@Entity
class MenuEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long = 0,
    val name: String,
    val price: Int,
    val image: String,
    @ManyToOne
    var store: StoreEntity
) {
    override fun toString(): String {
        return "MenuEntity(id=$id, name='$name', price=$price, image='$image', store=$store)"
    }
}
